packages/web-react/src/components/RadioField/__tests__/useRadioFieldStyleProps.test.ts

```diff
@@ -34,4 +36,25 @@ describe('useRadioFieldStyleProps', () => {
 
     expect(result.current.classProps.root).toBe('RadioField RadioField--item');
   });
+
+  it('should return field with danger', () => {
```

Comment:
Same as for the checkbox field, use `it.each`, please.

Reply:
done